packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -430,9 +420,12 @@ export async function getChatAuthState(cwAuth = AuthUtil.instance): Promise<Feat
         if (isValidCodeWhispererCoreConnection(currentConnection)) {
             state[Features.codewhispererCore] = AuthStates.connected
         }
-        if (isValidCodeWhispererChatConnection(currentConnection)) {
+        if (isValidAmazonQConnection(currentConnection)) {
             state[Features.codewhispererChat] = AuthStates.connected
         }
+        if (isValidAmazonQConnection(currentConnection)) {
```

Comment:
Why a separate if statement here? Is this intended or a merge issue?

Reply:
Nice catch. @aemada-aws can you fix this

Reply:
Hold on, sorry, I may be misunderstanding something

Reply:
It looks like `isValidCodeWhispererChatConnection` was removed, but unclear why. Let me take a closer look at this.